App/Redux/StartupRedux.ts

```diff
@@ -8,4 +10,26 @@ const actions = {
 
 export type StartupAction = ActionType<typeof actions>
 
+export type StartupState = {
+  started: boolean
```

Comment:
The startup action has always be fired when redux-persist rehydration is complete. Just adding some state to track when that happens so you can check for it in sagas.